downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -0,0 +1,27 @@
+[id="ref-general-inventory-variables"]
+
+= General variables
+
+[cols="50%,50%",options="header"]
+|====
+| *Variable* | *Description*
+| *`registry_password`* | Password credential for access to `registry_url`.
+
+Used for both `[automation controller]` and `[automation hub]` groups.
+
+Enter your Red Hat Registry Service Account credentials in `registry_username` and `registry_password` to link to the Red Hat container registry.
+
+The same applies in `registry_username`, but this is only if `registry_url` is registry.redhat.io.  
```

Comment:
I think this sentence can be removed since the description is duplicated in reistsry_username.